src/components/LoginSignUp/Login.js

```diff
@@ -126,12 +130,20 @@ const Login = ( ): Node => {
 
   const loginForm = (
     <>
+
```

Comment:
Please remove the blank line after the opening tag here. We should really have an eslint rule for this, but I haven't found one yet...